src/FondOfSpryker/Yves/GoogleTagManagerStoreConnector/Expander/StoreDataLayerExpander.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerStoreConnector\Expander;
+
+use FondOfSpryker\Shared\GoogleTagManagerStoreConnector\GoogleTagManagerStoreConnectorConstants;
+use FondOfSpryker\Yves\GoogleTagManagerStoreConnector\GoogleTagManagerStoreConnectorConfig;
+use Spryker\Shared\Kernel\Store;
+
+class StoreDataLayerExpander implements StoreDataLayerExpanderInterface
+{
+    /**
+     * @var \Spryker\Shared\Kernel\Store
+     */
+    protected $store;
+
+    /**
+     * @var \FondOfSpryker\Yves\GoogleTagManagerStoreConnector\GoogleTagManagerStoreConnectorConfig
+     */
+    protected $config;
+
+    /**
+     * @param \Spryker\Shared\Kernel\Store $store
+     * @param \FondOfSpryker\Yves\GoogleTagManagerStoreConnector\GoogleTagManagerStoreConnectorConfig $config
+     */
+    public function __construct(
+        Store $store,
```

Comment:
Use the a bridge of store client for getting current store information. --> https://github.com/spryker/store/blob/f55682a4502ce8afceef2439924a82b820ebe1f6/src/Spryker/Client/Store/StoreClientInterface.php#L22